config/config.php

```diff
@@ -108,6 +108,38 @@
 
 return [
 	'stable' => [
+		'29' => [
+			'100' => [
+				'latest' => '29.0.0',
+				'internalVersion' => '29.0.0.19',
+				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-29.0.0.zip',
+				'web' => 'https://docs.nextcloud.com/server/29/admin_manual/maintenance/upgrade.html',
+				'eol' => false,
+				'minPHPVersion' => '8.0',
+				'signature' => 'WYiOZ/o8pfhdiTsRgo2Y9deTYAhfe10Z0JrMOlqBBpNbGw8+Lzgkc4clWDM2TQ7D
+ZMXLZicwkUxueh3o7PZsbR7RHT2weQ8ajMz+wc87rCKVUiNzLm/7VXL9o5HXY7jw
+hhSbjS22PwyKXhrqoMyC5Zu7L60O+8W/LPUyYwnCV5IdL9BCXCa+Z284Ig95v8uX
+c/N0UlT+d4cQD/GxhIxDvNDtsV3pLnUjWwgSwjSbnsMu+4jL6AxtNO1ZWVdeg3X7
+o2+KFQzM84mURv71eGvzsCXziisZM1irTS7UZSYLlV8x3r/6xOoDFcnjDjquHLaQ
+NxJoXqJoiaG26rWzAT6yDQ==',
+			],
+		],
+		'28.0.4' => [
+			'100' => [
```

Comment:
We usually start with 30 percent on the first release of a major:
https://github.com/nextcloud-releases/updater_server/commit/6221cc2738827f7fb1194853d15b739f8c808274

Reply:
Updated! Thanks :)